pkgs/data/fonts/iosevka/package.json

```diff
@@ -1,28 +1,27 @@
 {
```

Comment:
Could you add a script that generates this automatically? Here's a working example:

```bash
#!/usr/bin/env bash

# Script to generate package.json needed to create the build environment.

set -eu -o pipefail
cd "$(dirname "${BASH_SOURCE[0]}")"

SRC="$(nix-build -A iosevka.src --no-out-link ../../../..)"
JQ_BIN="$(nix-build -A jq --no-out-link ../../../..)/bin/jq"


main() {
    jq '{
            "name": "iosevka-build-deps",
            "version": .version,
            "scripts": { "build": .scripts.build },
            "dependencies": .dependencies
        }' \
        "$SRC/package.json" \
        > package.json
}

jq() {
    command "$JQ_BIN" "$@"
}

main
```

Reply:
The latest push includes an `update.sh` script that does it.